packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -749,7 +749,7 @@ export class DeltaManager
 
                     // Socket.io error when we connect to wrong socket, or hit some multiplexing bug
                     if (!canRetryOnError(origError)) {
-                        const error = normalizeError(origError);
+                        const error = normalizeError(origError, { props: {...fatalConnectErrorProp} });
```

Comment:
What's the point of doing ... when you can just pass fatalConnectErrorProp?

Reply:
Just to make it more natural for the next person to add another prop.  I had it the other way earlier, can switch it back (or do your other idea of updating the API)